README.md

```diff
@@ -70,25 +70,31 @@ Initialize a project:
 ```sh
 $ mkdir pipcook-example && cd pipcook-example
 $ pipcook init
-$ cd pipcook-project
 ```
 
+### Show Cases
+If you are wondering what you can do in pipcook and where you can check your training logs and models, you could run pipboard first.
+```sh
+$ pipcook board
+```
+you will see a web page prompt in your browser. Now you can have a look at MNIST showcase on the home page and play around there. If you want to train a model to recognize MNIST handwritten digits by yourself, you could try the example below.
+
 ### Examples
```

Comment:
It should be removed for now, we have mentioned the examples at the previous section.

Reply:
@utkobe Hey, the examples section should be included the playground? One is to remove the title, or downgrade the title level to "#### Examples".

Reply:
Yes, that makes sense. I just modified it